spec/lib/percy/capybara/loaders/base_loader_spec.rb

```diff
@@ -14,6 +14,18 @@
       expect(resource.content).to include('Hello World!')
       expect(resource.sha).to eq(Digest::SHA256.hexdigest(resource.content))
     end
+    it 'includes an iframe resource' do
```

Comment:
What do you think about this: https://gist.github.com/christophemaximin/1016c25f437203c5902deded6635f771#file-diff-diff ?